### DISCUSSION TITLE
Dare ai riceventi la possibilità di creare utenti

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#4141] [Ideas]
Ciao, stavo pensando, anche sulla base dei feedback di alcuni nostri clienti, che potrebbe essere utile lasciar loro la possibilità di crearsi utenti interni in autonomia. Nelle opzioni degli utenti di tipo ricevente c'è la possibilità di fargli vedere/gestire la voce **Impostazioni**. Mi chiedevo se non fosse possibile mostrare anche la voce **Utenti**.
Grazie

**Comment by evilaliv3** [comment]
Grazie per il suggerimento @maelaborazioni 

In realtà la configruazione dei riceventi è qualcosa che richiede particolari attenzioni e configurazioni, motivo per cui ancora non siamo riusciti ad identificare una modalità in cui i riceventi si gestiscano le proprie utenze.

Piuttosto stiamo invece pensando di permettere di limitare i permessi di amminsitrazione così che si possa magari dare ai team lead un account di amministrazione limitato.

**Reply by maelaborazioni** [reply]
Grazie a te per la risposta @evilaliv3 e per le info su questa prossima _eventuale_ implementazione.
